lib/commandCenter/commandCenter.js

```diff
@@ -198,7 +199,7 @@ CommandCenter.prototype.setupUserCommand = function (modName, cmdName, cmd) {
 	const firstParameter = params.shift();
 
 	// Sanity check
-	if (firstParameter !== 'state') {
+	if (params.length !== 0 && firstParameter !== 'state') {
```

Comment:
Fixes a bug where in async user commands, you may not even need to use the state object